meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -17,7 +17,6 @@ SRC_URI:append:libc-musl = " \
     file://utils.c-disable-tilde-as-it-is-not-supported-by-musl.patch \
 "
 
-PACKAGECONFIG[sota-tools] = "-DBUILD_SOTA_TOOLS=ON ${GARAGE_SIGN_OPS},-DBUILD_SOTA_TOOLS=OFF,glib-2.0 ostree,"
```

Comment:
How will `garage-sign` be downloaded if `sota-tools` are removed?

Reply:
This same line/content is now available in the aktualizr recipe from meta-updater, so nothing changes.

Reply:
btw, we can remove `sota-tools`/native building of akltualizr at all. Effectively, what we need is just `garage-sign` which is downloaded by the python script while aktualizr is being built with `-DBUILD_SOTA_TOOLS=ON ` option. The other binaries built with this option are either replaced by fiopush/fiosync or non needed at all (`garage-deploy`).

Reply:
Indeed, we can follow up with that cleanup then.